model_analyzer/config/input/config_command_profile.py

```diff
@@ -1226,6 +1310,12 @@ def _autofill_values(self):
                     'triton_server_environment'] = model.triton_server_environment(
                     )
 
+            # triton docker args
+            if not model.triton_docker_args():
+                new_model['triton_docker_args'] = self.triton_docker_args
+            else:
+                new_model['triton_docker_args'] = model.triton_docker_args()
```

Comment:
This is global vs model-specific args?  But it is all or nothing, right?  If globally has 3 options set and then one is specified at the model level, only that one option is used?